src/main/scala/org/camunda/feel/api/FeelEngineBuilder.scala

```diff
@@ -74,3 +77,9 @@ case class FeelEngineBuilder private (
   )
 
 }
+
+object FeelEngineBuilder {
+
+  @javaStatic
```

Comment:
:thought_balloon: I don't see that the annotation is required. I can access the static method in the Java class without the annotation.

Reply:
Done